src/manipulation.js

```diff
@@ -172,9 +150,6 @@ function domManip( collection, args, callback, ignored ) {
 
 					// Keep references to cloned scripts for later restoration
 					if ( hasScripts ) {
-
-						// Support: Android <=4.0 only, PhantomJS 1 only
-						// push.apply(_, arraylike) throws on ancient WebKit
 						jQuery.merge( scripts, getAll( node, "script" ) );
```

Comment:
@mgol I was just going back over this PR and wondering about the loss of context here and before other `jQuery.merge` calls... _should_ we switch this to `push.apply( scripts, getAll( node, "script" ) )` or have a comment explaining why it avoids that pattern?

Reply:
Right... We no longer care about Android 4.0 or PhantomJS 1 but I remember using the `push.apply` approach may lead to issues like #4320. Perhaps a comment to that effect would be useful.

Reply:
I created #5298 to track this & other related uses of `push.apply` vs. `jQuery.merge` to consider.